### ISSUE TITLE
Make mobile-view responsive

### ISSUE THREAD

**Issue by Nuwanga94** [gmemstr/notdonemywebsite#9]
![vvdv](https://user-images.githubusercontent.com/36567586/46665737-55f58000-cbe2-11e8-8b14-8ec57333d6e4.PNG)

**Comment by gmemstr** [comment]
Agreed, definitely something that should be done - probably with some media queries to hide the menu in a side menu that slides out or the like. Open to other ideas, redoing the general UI/UX is also an option.
